### PR TITLE
Fix comment wording

### PR DESCRIPTION
**Pull request by dharmatech**
Changes comments in 9 files from the form:

```
// POST: Movies/Edit/5
// To protect from overposting attacks, enable the specific properties you want to bind to, for 
// more details, see http://go.microsoft.com/fwlink/?LinkId=317598.
```
to:
```
// POST: Movies/Edit/5
// To protect from overposting attacks, enable the specific properties you want to bind to.
// For more details, see http://go.microsoft.com/fwlink/?LinkId=317598.
```

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/Scaffolding?pullRequest=1349) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/Scaffolding?pullRequest=1349) before we can accept your contribution.<br/><br/>:x: dharmatech [sign now](https://cla.dotnetfoundation.org/dotnet/Scaffolding?pullRequest=1349)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/Scaffolding?pullRequest=1349) it.</sub>

File: src/VS.Web.CG.Mvc/Templates/ControllerGenerator/ApiControllerWithContext.cshtml [hunk 1]
```diff
@@ -56,8 +56,8 @@ namespace @Model.ControllerNamespace
         }
 
         // PUT: @routePrefix/5
-        // To protect from overposting attacks, enable the specific properties you want to bind to, for
-        // more details, see https://go.microsoft.com/fwlink/?linkid=2123754.
+        // To protect from overposting attacks, enable the specific properties you want to bind to.
```
Comment (Rick-Anderson) [inline src/VS.Web.CG.Mvc/Templates/ControllerGenerator/ApiControllerWithContext.cshtml:59]: // To protect from overposting attacks, see https://go.microsoft.com/fwlink/?linkid=2123754

properties you want to bind to is not used in API binding.
Ending the sentence with a period is correct grammar, but it makes it harder to copy/paste.

Generally we don't take PR's on slight improvements to comments, but fix this and it's worth taking.

File: src/VS.Web.CG.Mvc/Templates/ControllerGenerator/ApiControllerWithContext.cshtml [hunk 2]
```diff
@@ -88,8 +88,8 @@ namespace @Model.ControllerNamespace
         }
 
         // POST: @routePrefix
-        // To protect from overposting attacks, enable the specific properties you want to bind to, for
-        // more details, see https://go.microsoft.com/fwlink/?linkid=2123754.
+        // To protect from overposting attacks, enable the specific properties you want to bind to.
```
Comment (Rick-Anderson) [inline src/VS.Web.CG.Mvc/Templates/ControllerGenerator/ApiControllerWithContext.cshtml:91]: same problem here.

File: src/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap3/CreatePageModel.cshtml [hunk 3]
```diff
@@ -52,8 +52,8 @@ namespace @Model.NamespaceName
         [BindProperty]
         public @Model.ViewDataTypeShortName @Model.ViewDataTypeShortName { get; set; }
 
-        // To protect from overposting attacks, enable the specific properties you want to bind to, for
-        // more details, see https://aka.ms/RazorPagesCRUD.
+        // To protect from overposting attacks, enable the specific properties you want to bind to.
```
Comment (Rick-Anderson) [inline src/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap3/CreatePageModel.cshtml:55]: // To protect from overposting attacks, see https://aka.ms/RazorPagesCRUD.

RP doesn't use bind properties.

File: src/VS.Web.CG.Mvc/Templates/ControllerGenerator/ApiControllerWithContext.cshtml (hunk 1 above)
Comment (dharmatech) [reply src/VS.Web.CG.Mvc/Templates/ControllerGenerator/ApiControllerWithContext.cshtml]: Thanks for the feedback an corrections @Rick-Anderson ! I've updated the patch according to your recommendations.

File: src/VS.Web.CG.Mvc/Templates/ControllerGenerator/ApiControllerWithContext.cshtml (hunk 2 above)
Comment (dharmatech) [reply src/VS.Web.CG.Mvc/Templates/ControllerGenerator/ApiControllerWithContext.cshtml]: Updated

File: src/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap3/CreatePageModel.cshtml (hunk 3 above)
Comment (dharmatech) [reply src/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap3/CreatePageModel.cshtml]: Updated

File: src/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap4/CreatePageModel.cshtml [hunk 4]
```diff
@@ -52,8 +52,8 @@ namespace @Model.NamespaceName
         [BindProperty]
         public @Model.ViewDataTypeShortName @Model.ViewDataTypeShortName { get; set; }
 
-        // To protect from overposting attacks, enable the specific properties you want to bind to, for
-        // more details, see https://aka.ms/RazorPagesCRUD.
+        // To protect from overposting attacks, enable the specific properties you want to bind to.
```
[the remark this answers is not in the document]
Comment (dharmatech) [reply src/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap4/CreatePageModel.cshtml]: Updated

**Comment by deepchoudhery** [comment]
Thanks

**Pull request merged**